docs/usage/configuration.md

```diff
@@ -189,3 +189,14 @@ Modules added to this list are permitted in addition to the ones listed in `allo
 
 When `true`, allow all local modules to be imported, without being reported by the **forbidden-import** check.
 By default this option is `false`.
+
+### `mypy-options`
+
+A list of [configuration flags](https://mypy.readthedocs.io/en/stable/command_line.html) to be passed into mypy when performing the **static-type** check.
```

Comment:
instead of "configuration flags" refer to these as "command-line arguments".

Also, say "performing the static type checks" and provide a direct link to the "static type checks" section of the `checkers/index.html` page (you'll need to look up how to do cross-references with sphinx).